vm/src/tests/cairo_run_test.rs

```diff
@@ -1032,3 +1032,26 @@ fn divmod_igcdex_not_one() {
     let error_msg = "Operation failed: divmod(1, 340282366920938463463374607431768211457, 340282366920938463463374607431768211457), igcdex(340282366920938463463374607431768211457, 340282366920938463463374607431768211457) != 1";
     run_program_with_error(program_data.as_slice(), error_msg);
 }
+
+fn cairo_run_print_felt() {
```

Comment:
This is missing the `#[test]` attribute.

Reply:
Completly missed this sorry, will update